packages/core/src/properties/root.ts

```diff
@@ -180,6 +182,21 @@ export class Root extends Property {
 		return this.scenes.map((p) => p.getChild());
 	}
 
+
+	/**
+	 * Set the default scene ID associated with this root.
+	 * @hidden
+	 */
+	public _setDefaultScene(sceneId?: number): this {
```

Comment:
I think this method can be public, and just remove the method from the Document. Similar to `getExtras()` and `setExtras(...)` on the Root.